01 OperatorOverloading/OperatorOverloading.Model/Money.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.dbl;
+namespace OperatorOverloading.Model
+{
+    public class Money
+    {
+        private string _currency;
+
+        //constructor to assign values
+        public Money(double amount, string currency)
+        {
+            Amount = amount;
+            Currency = currency;
+        }
+        public Money() { }
+
+        public Money(String s)
+        {
+            Currency = s;
+        }
+
+        //properties 
+        public double Amount
+        {
+            get;
+            set;
+        }
+
+        public string Currency
+        {
+            get
+            {
+                return _currency;
+            }
+            set
+            {
+                if (string.IsNullOrWhiteSpace(value) || value.Length != 3)
+                    throw new Exception();
+                else
+                    _currency = value;
+            }
+        }
+
+        public double ConvertCurrency(double amount,string sourceCurrency, string targetCurrency)
```

Comment:
this method should be static. If you want an instance method then it will take only targetCurrency as source amount and currency is already their in the money object.